lambda/src/index.js

```diff
@@ -1,17 +1,37 @@
 import AWS from 'aws-sdk';
 import {loadFromS3} from './config'
-import repeat from 'lodash/repeat'
+import {getSheetsJson} from './google'
+
+let s3 = new AWS.S3();
 
 export function handler(event, context, callback) {
   console.log("Initialising.")
-  let creds = loadFromS3()
-  creds.then((creds)=>{
-    console.log(creds)
-    console.log(repeat('hello',10))
-    callback(null)
-  }).catch((error)=>{
-    console.error("Failed to get credentials",error)
-    callback(error)
+  let config = loadFromS3()
+  config.then((conf)=> {
+    let sheetJson = getSheetsJson(conf.keyFile, conf.sheetId).then(function(response) {
+      let parsed = response.map(parseRows);
+      let uploadParams = {
+        Bucket: conf.uploadBucket,
+        Key: "data.json",
+        Body: JSON.stringify(parsed)
+      };
+      let upload = s3.putObject(uploadParams, function(err, data) {
+        if (err) console.error("could not upload to s3", err);
+        else console.log('successfully uploaded to s3')
+        callback(null)
+      })
+    }).catch((error)=>{
+      console.error("Failed to get credentials",error)
+      callback(error)
+    })
   })
+}
 
+function parseRows(row) {
+  return {
+    question: row.question,
+    link: row.link,
+    keywords: row.keywords.replace(/ /g, "").split(","),
```

Comment:
This is useful, but also i think the `filter.js` plugin actually doesn't need it in an array. Umm, leave it as is anyway, I'll take a look and see what I would need to do, should be more useful this way anyway!